### PR TITLE
corrected schedule and file names

### PR DESCRIPTION
**Pull request by parispittman**

### CODE REVIEW

File: community/2017-events/12-contributor-summit/ContribSummitInformation.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Times in CST
 01:00pm - Sessions resume  
 04:30pm - Happy Hour onsite  
 
-View the complete session schedule [here](/kubernetes/community/community/2017-events/schedule).  
+View the complete session schedule [here](https://github.com/parispittman/community/blob/master/community/2017-events/12-contributor-summit/schedule.png).
```
Comment (chrislovecnm) [inline community/2017-events/12-contributor-summit/ContribSummitInformation.md]: Umm  this is you branch, use relative paths ;)

File: community/2017-events/12-contributor-summit/ContribSummitInformation.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Times in CST
 01:00pm - Sessions resume  
 04:30pm - Happy Hour onsite  
 
-View the complete session schedule [here](/kubernetes/community/community/2017-events/schedule).  
+View the complete session schedule [here](/community/community/2017-events/12-contributor-summit/schedule.png).
```
Comment (cblecker) [inline community/2017-events/12-contributor-summit/ContribSummitInformation.md]: one too many `/community`s in there

Comment (chrislovecnm) [reply community/2017-events/12-contributor-summit/ContribSummitInformation.md]: Should we do relative?

Comment (chrislovecnm) [reply community/2017-events/12-contributor-summit/ContribSummitInformation.md]: How do we not have link checker ... grumble

**Comment by chrislovecnm** [comment]
Paris is going to push another PR

File: community/2017-events/12-contributor-summit/ContribSummitInformation.md (hunk 2 above)
Comment (cblecker) [reply community/2017-events/12-contributor-summit/ContribSummitInformation.md]: if it's all in the same directory, relative also works. absolute is better if docs are moving around

Comment (chrislovecnm) [reply community/2017-events/12-contributor-summit/ContribSummitInformation.md]: It is in the same dir

**Pull request merged**